uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -151,7 +151,15 @@ public void addWorkAccount(@NonNull final ITestBroker broker,
 
     @Override
     public void forwardDeviceTimeForOneDay() {
-        Logger.i(TAG, "Forwarding Time For One Day on Google Device..");
+        forwardDeviceTime(86400);
+    }
+
+    /**
+     * @param seconds amount to advance time by
+     * Change the time on the device by advancing the clock by seconds.
+     */
+    public void forwardDeviceTime(int seconds) {
```

Comment:
I would suggest adding this directory to BaseSettings.java and then just override that here

Reply:
looks like you did that already...cool